integration/flink/examples/flink2-test-apps/build.gradle

```diff
@@ -0,0 +1,79 @@
+/*
+/* Copyright 2018-2025 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+plugins {
+    id 'java-library'
+    id 'jacoco'
+    id 'pmd'
+    id "com.adarshr.test-logger" version "4.0.0"
+    id 'com.diffplug.spotless' version '6.23.0'
+    id "com.github.johnrengelman.shadow" version "8.1.1"
+    id "com.github.davidmc24.gradle.plugin.avro" version "1.9.1"
+}
+
+repositories {
+    gradlePluginPortal()
+    mavenCentral()
+    maven { url "https://packages.confluent.io/maven/" }
+    mavenLocal()
+}
+
+group 'io.openlineage.flink'
+
+shadowJar {
+    zip64 true
+}
+
+ext {
+    flinkVersion = project.getProperty('flink.version')
+    flinkVersionShort = flinkVersion.substring(0,4)
+}
+
+dependencies {
+    configurations.all {
+        exclude module: 'slf4j-log4j12'
+        exclude module: 'log4j'
+    }
+
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter:5.10.2"
+    testCompileOnly 'org.codehaus.groovy:groovy-all:3.0.20'
+    testImplementation 'org.spockframework:spock-core:2.3-groovy-4.0'
+    implementation 'org.awaitility:awaitility:4.2.0'
+    implementation 'org.apache.httpcomponents.client5:httpclient5:5.3.1'
+    implementation "com.typesafe:config:1.4.3"
```

Comment:
nit: you can try adding newest version of dependencies if you're adding those files anyway.